cypress/e2e/business.management.cy.tsx

```diff
@@ -60,23 +60,46 @@ describe('Business Management Page ', () => {
 
         //Enter product name and description
         cy.get('input[name="name"]').type('Product 1')
+        cy.get('textarea[name="description"]').click()
         cy.get('textarea[name="description"]').type('This is description for Product 1')
+        cy.get('input[type="file"]').attachFile('2DS_Attributes.png');
+        //   cy.contains('2DS_Attributes.png');
+        cy.wait(500)
         cy.get('input[type="submit"]').click()
 
+
         //Go back to products page and verify we see our product
         cy.get('a[href*="/business/offering?name=Automation Test"]').click()
         cy.get('input[name="password"]').type('0/k0t2qZK/Ak')
         cy.get('input[type="submit"]').click()
         cy.get('td').contains('Product 1')
 
+        //Go to defects page for Boston University
```

Comment:
Yeah I did mean to leave it commented out. I was trying to write in some tests to ensure I could navigate to Boston University and check the defects there, but I kept running into issues. Figured there are a lot of final changes coming so I wanted to keep what I had just in case I have some time to pick it up and create a full end to end flow for a test company. If not, we can just rely on the manual checks we’ve written and I’ll delete the verifications that aren’t being executed.